### ISSUE TITLE
Italian translation

### ISSUE THREAD

**Issue by AlexTux87** [rescuezilla/rescuezilla#154]
Attached are the three files translated into Italian
[it-translation.zip](https://github.com/rescuezilla/rescuezilla/files/5683910/it-translation.zip)

**Comment by shasheene** [comment]
Hi @AlexTux87,

Thanks for the translation!

<a href="https://user-images.githubusercontent.com/1414379/102002752-ada64d80-3d4f-11eb-8ce6-f6a93a819eaa.png"><img width=300 src="https://user-images.githubusercontent.com/1414379/102002752-ada64d80-3d4f-11eb-8ce6-f6a93a819eaa.png"></a> <a href="https://user-images.githubusercontent.com/1414379/102002756-af701100-3d4f-11eb-87e0-ea20a76785a8.png"><img width=300 src=""></a> <a href=""><img width=300 src="https://user-images.githubusercontent.com/1414379/102002756-af701100-3d4f-11eb-87e0-ea20a76785a8.png"></a> <a href="https://user-images.githubusercontent.com/1414379/102002758-b008a780-3d4f-11eb-896b-60d90671632a.png"><img width=300 src="https://user-images.githubusercontent.com/1414379/102002758-b008a780-3d4f-11eb-896b-60d90671632a.png"></a> <a href="https://user-images.githubusercontent.com/1414379/102002760-b0a13e00-3d4f-11eb-8bfb-0b96feb076f1.png"><img width=300 src="https://user-images.githubusercontent.com/1414379/102002760-b0a13e00-3d4f-11eb-8bfb-0b96feb076f1.png"></a> <a href="https://user-images.githubusercontent.com/1414379/102002761-b139d480-3d4f-11eb-8c5a-a2f15c05b4ce.png"><img width=300 src="https://user-images.githubusercontent.com/1414379/102002761-b139d480-3d4f-11eb-8c5a-a2f15c05b4ce.png"></a> 

I have been spending too much time working on Rescuezilla recently, so I was intending the next versioned Rescuezilla release (v2.2) to be released **_after April 2021_**. Understandably this is way too far away for the Italian-speaking users who will want your translation immediately. I don't want to make a v2.1.1 release with the only change being your new Italian translation, because I just released v2.1 yesterday.

**Because of this, I have created a custom build: [rescuezilla-2.1-2-g4d9fe8fd-italiano-64bit.groovy.iso](https://drive.google.com/u/0/uc?id=1Sk-BhHtOZrQodKhqah3fN4CGj-vXcMTS&export=download)**

**I have selected the standard Italian keyboard layout for your translation. Let me know if this is not sufficient.**

Here are the checksums:

```
sha256sum:
2cd224e609214dc412f8c95e2585d0d6aab85265bd4ff122b06b0d300a7798c1  rescuezilla-2.1-2-g4d9fe8fd-italiano-64bit.groovy.iso

sha1sum:
092ea87edffb43ee4be9c91041536b1e2e133f07  rescuezilla-2.1-2-g4d9fe8fd-italiano-64bit.groovy.iso

md5sum:
f1e5164befa36ca04864f84c93dd4355  rescuezilla-2.1-2-g4d9fe8fd-italiano-64bit.groovy.iso
```

Please try out this version and let me know how you go. Feel free to distribute the build widely. (It's literally the exactly the same as Rescuezilla v2.1 released yesterday but with your Italian translation)

Download: [rescuezilla-2.1-2-g4d9fe8fd-italiano-64bit.groovy.iso](https://drive.google.com/u/0/uc?id=1Sk-BhHtOZrQodKhqah3fN4CGj-vXcMTS&export=download)

**Comment by shasheene** [comment]
@AlexTux87,

I forgot to tell you, I recently released [Rescuezilla v2.1.1](https://github.com/rescuezilla/rescuezilla/releases/latest) to fix a bug introduced a day earlier.

Your Italian translation is now part of the official Rescuezilla release. I have removed the link in the previous comment to the temporary release I made earlier.

Let me know if you have any issues!
